css/about.css

```diff
@@ -0,0 +1,178 @@
+@import url("https://fonts.googleapis.com/css2?family=Paytone+One&family=Rubik+Mono+One&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.text2 {
+  background-color: gray;
+  text-align: center;
+  padding: 15px 5px;
+  font-family: "Lato", sans-serif;
+}
+
+.contact {
+  text-align: center;
+  color: orangered;
+  font-size: 16px;
```

Comment:
- [ ] [OPTIONAL] Consider using relative unit for font sizes. Absolute units such as `px`, `pt`, and `in` are fixed and do not change size across page presentations.<br/> Relative units, on the other hand, are not fixed. Instead, they’re sized based on a parent element or on the size of the viewport (i.e., the visible page area). This allows them to scale smoothly up and down with other page content as the screen size changes.

So, instead of:
```css
font-size: 16px;
```
You can do:
```css
font-size: 1rem;
```